cpp/oneapi/dal/algo/svm/detail/kernel_function.cpp

```diff
@@ -106,25 +117,29 @@ class daal_interop_rbf_kernel_impl : public kernel_function_impl {
 public:
     daal_interop_rbf_kernel_impl(double sigma) : sigma_(sigma) {}
 
-    daal_kf_t get_daal_kernel_function() override {
-        constexpr daal_rbf_kernel::Method daal_method = get_daal_method();
-        auto alg = new daal_rbf_kernel::Batch<Float, daal_method>;
-        alg->parameter.sigma = sigma_;
-        return daal_kf_t(alg);
+    daal_kf_t get_daal_kernel_function(bool is_dense) override {
+        if (is_dense) {
+            constexpr daal_rbf_kernel::Method daal_method = get_daal_method(true);
+            auto alg = new daal_rbf_kernel::Batch<Float, daal_method>;
+            alg->parameter.sigma = sigma_;
+            return daal_kf_t(alg);
+        }
+        else {
+            constexpr daal_rbf_kernel::Method daal_method = get_daal_method(false);
```

Comment:
better separate method: `get_daal_dense_method` and `get_daal_csr_method`

Reply:
Ok, it was fixed